src/main/scala/cromwell/engine/WorkflowManager.scala

```diff
@@ -58,6 +60,13 @@ trait ActorWorkflowManager extends WorkflowManager {
   }
 
   override def workflowStatus(id: WorkflowId): Option[WorkflowState] = workflowById(id) map {x => workflowState(x)}
+  override def workflowOutputs(id: WorkflowId): Map[FullyQualifiedName, WdlValue] = {
+    val outputs = workflowById(id) map {_ ! GetOutputs}
```

Comment:
I think this is broken, the `map` is returning `Unit` and then `Unit` is being cast to `Option[Set[SymbolStoreEntry]]`.